doc/diagnostics/diag_run.rst

```diff
@@ -97,22 +110,32 @@ Each package can be run/configured from the command line using the wrapper scrip
   -i2, --input-dir2=DIR                         Specify the directory where the control case history files are located (OPTIONAL).
                                                 Default is --input-dir=/projects/NS2345K/noresm/cases
   -o, --output-dir=DIR                          Specify the directory where the package(s) the climatology and time-series files should be stored (OPTIONAL).
-                                                Default is --output-dir=/projects/NS2345K/noresm_diagnostics/out/$USER
+                                                Default is --output-dir=/projects/NS2345K/diagnostics/noresm/out/yanchun
   -p, --passive-mode                            Run the script in passive mode: the diagnostic script will be configured but not executed (OPTIONAL).
   -t, --type=TYPE                               Specify climatology or time series diagnostics (OPTIONAL): valid options are --type=climo and --type=time_series.
                                                 Default is to run both. Note that the time series are computed over the entire simulation.
   -w, --web-dir=DIR                             Specify the directory where the html should be published (OPTIONAL).
-                                                Default is --web-dir=/projects/NS2345K/www/noresm_diagnostics
+                                                Default is --web-dir=/projects/NS2345K/www/diagnostics/noresm/yanchun
```

Comment:
I think "yanchun" should be replaced by "$USER". Not sure if this is the actual default, but it should be.

Reply:
@TomasTorsvik 
Hi,
I'll close this pull request and make a new with the corrections you suggested